.ruff.toml

```diff
@@ -0,0 +1,32 @@
+select = [
+    "E",    # pycodestyle
+    "W",    # pycodestyle
+    "F",    # pyflake
+    "I",    # isort
+    "B",    # flake8-bugbear
+    "UP",   # pyupgrade
```

Comment:
It's nice to also include `C4` in addition to `UP`, since `C4` handles "unnecessary comprehensions" which was formerly handled by pyupgrade https://beta.ruff.rs/docs/rules/#flake8-comprehensions-c4

Reply:
Added. Thanks! Didn't know about that.